code/Generators/DocBlockGenerator.php

```diff
@@ -86,11 +88,17 @@ public function getExistingTagComments()
     }
 
     /**
-     * @return DocBlock\Tag[]
+     * @return ArrayList|DocBlock\Tag[]
      */
     public function getGeneratedTags()
```

Comment:
Do you expect the generator to be extended? The idea is good, but I doubt it adds value?